### PR TITLE
Jezra/dev

### PR DESCRIPTION
**Pull request by nanojezra**

### CODE REVIEW

**Comment by nanojezra** [comment]
please ignore all but the last merge. I merged main into my dev environment.

File: app/templates/authenticated/writing-groups/show/admin.hbs [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {{/if}}
   </ul>
   <div class="grid-wrapper">
-    {{group-users-list group=group listType='all' class="nw-card g3-col1 g3-row1 g3-hspan2 g3-vspan3 g2-col1 g2-row1 g2-hspan1 g2-vspan3"}}
+    {{group-users-list group=group isEditable=true listType='all' class="nw-card g3-col1 g3-row1 g3-hspan2 g3-vspan3 g2-col1 g2-row1 g2-hspan1 g2-vspan3"}}
```
Comment (uharin) [inline app/templates/authenticated/writing-groups/show/admin.hbs:9]: Just checking: did you intend for this to be a hardcoded boolean?

Comment (nanojezra) [reply app/templates/authenticated/writing-groups/show/admin.hbs:9]: absolutely. that is the "fix"

Comment (uharin) [reply app/templates/authenticated/writing-groups/show/admin.hbs:9]: Ok, good deal. I had a feeling that might be it. When in doubt, put a boolean on it!

**Review by uharin — APPROVED** [review]

File: app/templates/authenticated/writing-groups/show/admin.hbs (hunk 1 above)
Comment (nanojezra) [reply app/templates/authenticated/writing-groups/show/admin.hbs:9]: when a group admin views that component, isEditable must be true in order for the buttons to appear

**Pull request merged**